src/components/screens/ExperimentCreator.vue

```diff
@@ -0,0 +1,1348 @@
+                        <li>structure of biological or technical replicates (if applicable).</li>
+                      </ul>
+                    </div>
+                  </div>
+                  <div class="mavedb-wizard-content field">
+                    <TabView>
+                      <TabPanel header="Edit">
+                        <span class="p-float-label">
+                          <Textarea v-model="methodText" :id="$scopedId('input-methodText')" rows="10" />
+                          <label :for="$scopedId('input-methodText')">Methods</label>
+                        </span>
+                      </TabPanel>
+                      <TabPanel header="Preview">
+                        <div class="mavedb-wizard-rendered-markdown" v-html="markdownToHtml(methodText)"></div>
+                      </TabPanel>
+                    </TabView>
+                    <span v-if="validationErrors.methodText" class="mave-field-error">{{ validationErrors.methodText }}</span>
+                  </div>
+                </div>
+                <div class="mavedb-wizard-row">
+                  <div class="mavedb-wizard-help">
+                    <label>
+                      Digital Object Identifier
+                    </label>
+                    <div class="mavedb-help-small">
+                      Preprints or publications that are not indexed by PubMed can be included via the DOI field.
+                    </div>
+                  </div>
+                  <div class="mavedb-wizard-content field">
+                    <span class="p-float-label">
+                      <Chips
+                          ref="doiIdentifiersInput"
+                          v-model="doiIdentifiers"
+                          :id="$scopedId('input-doiIdentifiers')"
+                          :addOnBlur="true"
+                          :allowDuplicate="false"
+                          @add="acceptNewDoiIdentifier"
+                          @keyup.escape="clearDoiIdentifierSearch"
+                        >
+                          <template #chip="slotProps">
+                            <div>
+                                <span>{{ slotProps.value.identifier }}</span>
+                            </div>
+                        </template>
+                      </Chips>
+                      <label :for="$scopedId('input-doiIdentifiers')">DOIs</label>
+                    </span>
+                    <span v-if="validationErrors.doiIdentifiers" class="mave-field-error">{{validationErrors.doiIdentifiers}}</span>
+                  </div>
+                </div>
+                <div v-if="itemStatus == 'NotLoaded' || this.item.private == true">
+                  <div class="mavedb-wizard-row">
+                    <div class="mavedb-wizard-help">
+                      <label>
+                        Contributors who may edit this experiment. Enter each contributor's
```

Comment:
Let's make the help text for contributors the same on this page as already exists on the score set page.

Reply:
Both of them are nearly the same, except score set is "Contributors who may edit this score set" and experiment is "Contributors who may edit this experiment". I think edit this experiment is more suitable except they cannot edit this experiment and only can edit related score set.